ThickGalaxy/Spirals.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.axes3d as p3
+
+
+
+def vel(r):
+    """
+    vel(r) represents the velocity as a function of radius and thus defines the rotation curve.
+     To obtain a spiral shape, we make the value of this function larger at smaller radii.
+
+    In this case, for radii (ar) less than or equal to 5 units, v=1.4(ar)exp(-ar/20).
+    For ar between 5 and 6, v ranges between 5.0 and 1.4(5)exp(-5/20).
+    For ar greater than or equal to 6, v is a constant 5.
+
+    :param r: radius
+    :return: velocity
+    """
+
+    ar = np.abs(r)
+    if np.any(ar <= 5.0):
+        v = 1.4*ar*np.exp(-ar/20.0)
+    elif np.any(ar > 5.0) and np.any(ar < 6.0):
+        max = 1.4*5*np.exp(-5.0/20.0)
+        min = 5.0
+        m = (max - min)/(-1.0)
+        v = m*(ar - 5.0) + max
+    else:
+        v = 5.0
+
+    return v
+
+
```

Comment:
There should only be two blank lines between each function according to pep8. It's helpful to open the script in pycharm as it will show you all such pep8 errors.